families.py

```diff
@@ -217,6 +351,60 @@ def idistinv(self, u, n):
                 
         return x
     
+    
+    
+    def fidistinv_jacobi_setup(self, n, data):
+        ns = np.arange(len(data), n+1)
+        
+        for q in range(ns.size):
+            
+            nn = ns[q]
+            
+            if nn == 0:
+                ug = np.array([0,1])
+            else:
+                xg,wg = self.gauss_quadrature(nn)
+                ug = self.idist(xg, nn)
+                ug = np.insert(ug, [0,ug.size], [0,1])
+                
+            exps = np.array([self.beta/(self.beta+1), self.alpha/(self.alpha+1)])
+            ug,exponents = fidistinv_setup_helper1(ug,exps)
+            
+            idistinv = lambda u: self.idistinv(u,nn)
+            data.append(fidistinv_setup_helper2(ug, idistinv, exponents, 10))
+            
+        return data
+    
+    def fidistinv(self, u, n):
+        import pickle
+        from pathlib import Path
```

Comment:
Let's put all imports at the top, especially for standard modules. See the second bullet of
[https://www.python.org/dev/peps/pep-0008/#imports](url)

Though being a slave to the PEP's is probably not necessary, I think we should try to adhere mostly to them. (Yes, some existing code does function-local imports...which should be changed as well.)